Source/common/SNTConfigurator.m

```diff
@@ -494,6 +496,10 @@ - (BOOL)enableBadSignatureProtection {
   return number ? [number boolValue] : NO;
 }
 
+- (NSString *)aboutText {
```

Comment:
Please add the method:

```
+ (NSSet *)keyPathsForValuesAffectingAboutText {
  return [self configStateSet];
}
```

near the other related methods further up (around line 261). This will ensure that if the config value changes while Santa.app is running it will still pick it up.

Reply:
see f0814d6